### DISCUSSION TITLE
Unable to customize PPT for Mac ribbon & toolbar with Instrumenta

### DISCUSSION THREAD

**Discussion by mueezkq** [iappyx/Instrumenta#38] [Q&A] [answered]
Has anyone experienced issues with customizing the ribbon and toolbar in PPT for Mac after installing Instrumenta? I can modify existing default PPT commands and tabs, but can't create any customizations and certainly can't add any Instrumenta commands to anything.

I've had no issues with Instrumenta and PPT on Windows, have been using it daily there for over 6 months and loving it. On macOS however, this has been an issue since October 2023. Updating PPT and Instrumenta releases haven't resolved the issue.

**Comment by mueezkq** [comment]
I'm also unable to make any edits to the Instrumenta tab itself

**Comment by iappyx** [comment, marked as answer]
Hi @mueezkq, I am afraid this is a limitation of PowerPoint for Mac that does not allow you to customize custom ribbons like Instrumenta. The only thing you can do is switch from "Pro-mode" to "Review mode" in Instrumenta settings or you can include Instrumenta commands in the Quick Access Bar.

**Reply by mueezkq** [reply]
Thanks for the reply @iappyx! Adding Instrumenta commands to the Quick Access Bar has also never worked for me on macOS. I can add instruments commands in the settings but when I click save they don't show up in the quick access bar. 

![CleanShot 2025-02-12 at 20 11 20@2x](https://github.com/user-attachments/assets/85367689-6620-4dd3-b136-ca7d4d2eef5e)
![CleanShot 2025-02-12 at 20 11 37@2x](https://github.com/user-attachments/assets/c1c7eecb-5f01-4250-afae-f134992496eb)
